tests/test_samples_rest.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for REST samples."""
+
+import unittest
+
+import dotenv
+from requests import HTTPError, JSONDecodeError
+
+from samples.rest.BingWebSearchV7 import web_search_basic
+
+# from samples.rest.BingAutosuggestV7 import
+
+
+class RESTSamplesTest(unittest.TestCase):
+    """Class for testing REST samples."""
+
+    def setUp(self):
+        self.env = dotenv.dotenv_values()
+
+    def test_web_search_basic(self):
+        """Test the basic REST call to Web Search API"""
+        response = web_search_basic(subscription_key=self.env.get(
+            'BING_SEARCH_V7_WEB_SEARCH_SUBSCRIPTION_KEY'))
+
+        self.assertEqual(response.status_code, 200)
+
+    def test_web_search_response_is_json(self):
```

Comment:
Can we test a sample query, validate the _type, and that web results are returned?

Reply:
Done